### DISCUSSION TITLE
Deprecation of `pkg`

### DISCUSSION THREAD

**Discussion by NLueg** [vercel/community#5353] [Help] [answered]
### Summary

I just saw that you deprecated pkg with a notice, that there are several forks out there.
This of course is no problem, but I had a few issues with the forks. After Googling some of it I always found issues in your repo that discussed it.
Unfortunately, you also deactivated the issues tab that's why a lot of documentation is missing and makes it hard to use any of the forks.

Long story short, is it possible to reactivate the issues for the repo [vercel/pkg](https://github.com/vercel/pkg)?

### Steps to Reproduce

Issues tab is missing at [vercel/pkg](https://github.com/vercel/pkg)

**Comment by leerob** [comment]
I would recommend opening issues on the forks – the repo is now deprecated/archived 🙏

**Reply by NLueg** [reply]
Thanks for the fast response! For sure, new issues shouldn't be created there. But there are / were more than 1000 issues (at least regarding to the webarchive) that are closed and answered by maintainer. They are now lost and can't be viewed anymore as forks don't inherit the issues.

The issues are the only source for different issues I came up with. And I can only say that as Google still shows the short description for the issues but I only see a `404` when opening them (for example https://github.com/vercel/pkg/issues/1866)

**Reply by leerob** [reply, marked as answer]
Unarchived, reopened issues for history, and archived again 👍

**Reply by NLueg** [reply]
Thank you so much!
